Formula/hyperkit.rb

```diff
@@ -11,9 +11,8 @@ class Hyperkit < Formula
     sha256 cellar: :any_skip_relocation, high_sierra: "f057fe7b3856421d0fdf1df3d9981e0729ee77c27ed3d4cb918e9523f7d5d9be"
   end
 
-  depends_on "aspcud" => :build
   depends_on "ocaml" => :build
-  depends_on "opam" => :build
+  depends_on "opam" => [">=2.0", :build]
```

Comment:
```suggestion
  depends_on "opam" => :build
  depends_on "pkg-config" => :build
```
Specifying the version doesn't do anything.

Reply:
Thanks didn't know that :)